test/t8n/t8n.cpp

```diff
@@ -103,11 +103,16 @@ int main(int argc, const char* argv[])
                     tx.chain_id = chain_id;
 
                     auto res = state::transition(state, block, tx, rev, vm);
+
+                    const auto tx_hash = hex0x(keccak256(rlp::encode(tx)));
+                    if (j_txs[i].contains("hash") && j_txs[i]["hash"] != tx_hash)
```

Comment:
Comparing string is risky. They may differ by "0x" prefix or hex digit case. Keep them as `hash256` type and only convert to string for printing.

Reply:
Done